framework/include/variables/MooseVariableFV.h

```diff
@@ -365,6 +373,17 @@ class MooseVariableFV : public MooseVariableField<OutputType>
 
   /// Holder for all the data associated with the neighbor element
   std::unique_ptr<MooseVariableDataFV<OutputType>> _neighbor_data;
+
+private:
+  /**
+   * Check and see whether the AD indexing scheme is compatible with the scaling factors
```

Comment:
You may have explained this in some discussion, but maybe the basic reasoning should be captured (a comment here would probably suffice).

Reply:
We should at least make a github issue to track progress making this work with scaling.  I expect that this will be VERY important sooner rather than later.  I know with pronghorn at least, scaling is basically required in order to get acceptable convergence.

Reply:
I've added this to the to-do list for #15732 and beefed up the doxygen string here